assignments/lambda-classes.js

```diff
@@ -1 +1,267 @@
+// instructorThree.demo('JavaScript');
+
+// Go below student test data for .grade() test
+
+const student1 = new Student({
+    'name': 'Stud1',
+    'age': 10,
+    'location': 'PlaceOne',
+    'gender': 'A',
+    'previousBackground': 'BGOne',
+    'className': 'CSOne',
+    'favSubjects': ['Sub1', 'Sub2', 'Sub3'],
+    'grade': 50,
+});
+
+const student2 = new Student({
+    'name': 'Stud2',
+    'age': 12,
+    'location': 'PlaceTwo',
+    'gender': 'B',
+    'previousBackground': 'BGTwo',
+    'className': 'UITwo',
+    'favSubjects': ['Subject1', 'Subject2', 'Subject3'],
+    'grade': 60,
+});
+
+const student3 = new Student({
+    'name': 'Stud3',
+    'age': 14,
+    'location': 'PlaceThree',
+    'gender': 'C',
+    'previousBackground': 'BGThree',
+    'className': 'DataThree',
+    'favSubjects': ['SUBJECT1', 'SUBJECT2', 'SUBJECT3'],
+    'grade': 90,
+});
+
+// instructorOne.grade(student1, 'Java');
+// instructorTwo.grade(student2, 'Python');
+// instructorThree.grade(student3, 'JavaScript');
+
+// console.log(student1);
+// console.log(student2);
+// console.log(student3);
+
+// student1.listsSubjects();
+// student2.listsSubjects();
+// student3.listsSubjects();
+
+// student1.PRAssignemnt('Java');
+// student2.PRAssignemnt('Python');
+// student3.PRAssignemnt('JavaScript');
+
+const pmOne = new ProjectManager({
+    'name': 'PM1',
+    'age': 201,
+    'location': 'home1',
+    'gender': 'a',
+    'specialty': 'Thing1',
+    'favLanguage': 'Lang1',
+    'catchPhrase': 'Phrase1',
+    'gradClassName': 'CS1',
+    'favInstructor': 'Instr1',
+});
+
+const pmTwo = new ProjectManager({
+    'name': 'PM2',
+    'age': 202,
+    'location': 'home2',
+    'gender': 'b',
+    'specialty': 'Thing2',
+    'favLanguage': 'Lang2',
+    'catchPhrase': 'Phrase2',
+    'gradClassName': 'CS2',
+    'favInstructor': 'Instr2',
+});
+
+const pmThree = new ProjectManager({
+    'name': 'PM3',
+    'age': 203,
+    'location': 'home3',
+    'gender': 'c',
+    'specialty': 'Thing3',
+    'favLanguage': 'Lang3',
+    'catchPhrase': 'Phrase3',
+    'gradClassName': 'CS3',
+    'favInstructor': 'Instr3',
+});
+
+// console.log(pmOne);
+// console.log(pmTwo);
+// console.log(pmThree);
+
+// pmOne.standUp('web1');
+// pmTwo.standUp('web2');
+// pmThree.standUp('web3');
+
+// pmOne.debugsCode(student1, 'Java');
+// pmOne.debugsCode(student2, 'Python');
+// pmOne.debugsCode(student3, 'JavaScript');
+
+
+// Checking grade functionality, STRETCH GOAL
+
+
+// console.log(student1.grade);
+// console.log(student2.grade);
+// console.log(student3.grade);
+
+// instructorOne.gradeStudent(student1);
+// console.log(student1.grade);
+// instructorOne.gradeStudent(student1);
+// console.log(student1.grade);
+// student1.graduate();
+
+// pmThree.gradeStudent(student3);
+// console.log(student3.grade);
+// pmThree.gradeStudent(student3);
+// console.log(student3.grade);
+// pmThree.gradeStudent(student3);
+// console.log(student3.grade);
+// student3.graduate();
```

Comment:
Lots of testing, I love it! Keep this up, it just gets more important as we go on.